internal/cmd/root.go

```diff
@@ -45,6 +46,13 @@ func NewRootCmd(version, date string, p *print.Printer) *cobra.Command {
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
 			p.Cmd = cmd
 			p.Verbosity = print.Level(globalflags.Parse(p, cmd).Verbosity)
+
+			argsString := print.BuildDebugStrFromSlice(os.Args)
+			p.Debug(print.DebugLevel, "arguments entered to the CLI: %s", argsString)
```

Comment:
```suggestion
			p.Debug(print.DebugLevel, "arguments: %s", argsString)
```

To reduce noise on the logs